helper-cli/src/main/kotlin/commands/ConvertOrtFileCommand.kt

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
It took me a while to figure out that "convert" in this context means the transformation to another file format. Maybe this should be explicitly mentioned in the commit message.

Reply:
I've applied @sschuberth proposal to use the transform command.